scraper.rb

```diff
@@ -32,21 +38,29 @@ def datefrom(date)
   body   = mp.css('div.koru-side-holder')
 
   data = {
-    id: mp_url.split("/")[-2],
-    name: body.css("div[role='main']").css('h1').inner_text,
-    sort_name: link.inner_text.strip,
-    party: body.css('.informaltable td')[1].inner_text,
-    area:  body.css('.informaltable td')[0].inner_text,
-    photo: body.css('.document-panel__img img/@src').last.text,
-    email: body.css('a.square-btn').attr('href').inner_text.gsub('mailto:',''),
-    facebook: body.css('div.related-links__item a[@href*="facebook"]/@href').text,
-    twitter:  body.css('div.related-links__item a[@href*="twitter"]/@href').text,
-    term: 51,
-    source: mp_url,
+    id:         mp_url.split("/")[-2],
+    name:       body.css("div[role='main']").css('h1').inner_text,
+    sort_name:  link.inner_text.strip,
+    photo:      body.css('.document-panel__img img/@src').last.text,
+    email:      body.css('a.square-btn').attr('href').inner_text.gsub('mailto:',''),
+    facebook:   body.css('div.related-links__item a[@href*="facebook"]/@href').text,
+    twitter:    body.css('div.related-links__item a[@href*="twitter"]/@href').text,
+    term:       TERM,
+    source:     mp_url
   }
-  data[:photo].prepend(base_url) unless data[:photo].nil? or data[:photo].empty?
+
+  data[:photo] = URI.join(mp_url, data[:photo]).to_s unless data[:photo].to_s.empty?
+
+  body.css('.informaltable tr').each do |row|
```

Comment:
I thought about this as well... do we want to have this data? (I guess we do)
If so, should I detect the change in term and change the `:term` field accordingly?